core/controllers/profile_test.py

```diff
@@ -964,6 +964,29 @@ def test_get_delete_account_page(self):
         self.assertIn(b'<oppia-root></oppia-root>', response.body)
 
 
+class SubscribeToAndroidListTests(test_utils.GenericTestBase):
+
+    def test_put_function(self):
+        # The GET function should not throw any error and should return status
```

Comment:
Seems weird that you're talking about a GET function in a PUT function -- at first, I thought this comment referred to the following line.

I think it's OK to just omit this if you're talking about the absence of a get() handler?

Reply:
Done, removed.